cdap-ui/app/cdap/components/ConfigurationGroup/index.tsx

```diff
@@ -38,13 +38,16 @@ const styles = (theme): StyleRules => {
     },
     groupTitle: {
       marginBottom: '15px',
+      marginLeft: 10,
+      marginRight: 15,
     },
     h2Title: {
-      ...h2Styles(theme),
-      marginBottom: 0,
+      ...h2Styles(theme).root,
+      marginBottom: 5,
     },
     groupSubTitle: {
       color: theme.palette.grey[200],
+      display: 'inline-block',
```

Comment:
why is this needed? When I see in the studio, the description is under the group title since the title is a block element.